src/components/SteppedProgress.js

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+import Step from './Step.js';
+import PropTypes from 'prop-types';
+
+SteppedProgress.propTypes = {
+  step: PropTypes.number
+};
+
+SteppedProgress.defaultProps = {
+  step: 1
+};
+
+export default function SteppedProgress({ step }) {
+  const getColorNeonBlue = ({ theme }) => theme.color.neonBlue;
+
+  return (
+    <Container>
+      <Progress></Progress>
```

Comment:
You can replace it with a self-closing tag: <Progress />

Reply:
Thank you for the hint!